eipw-lint/src/lints/markdown/no_backticks.rs

```diff
@@ -0,0 +1,130 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+ use annotate_snippets::snippet::{Annotation, AnnotationType, Slice, Snippet};
+
+ use comrak::nodes::{Ast, NodeCode, NodeCodeBlock, NodeHtmlBlock, NodeLink};
+ 
+ use crate::lints::{Context, Error, Lint};
+ use crate::tree::{self, Next, TraverseExt};
+ 
+ use ::regex::Regex;
+ 
+ use serde::{Deserialize, Serialize};
+ 
+ // use std::collections::HashSet;
+ use std::fmt::{Debug, Display};
+ 
+ #[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+ #[serde(transparent)]
+ pub struct NoBackticks<S>(pub S);
+ 
+ impl<S> Lint for NoBackticks<S>
+ where
+     S: Display + Debug + AsRef<str>,
+ {
+     fn lint<'a>(&self, slug: &'a str, ctx: &Context<'a, '_>) -> Result<(), Error> {
+         let pattern = self.0.as_ref();
+         let re = Regex::new(pattern).map_err(Error::custom)?;
+         let mut visitor = Visitor {
+             ctx,
+             re,
+             pattern,
+             slug,
+             found_backticks: false,
```

Comment:
I don't see where this member is used?

Reply:
Hi, I'm sorry for the late response.
What member are you referring to?

Reply:
`found_backticks`

Reply:
It seems like it gets set to true, but its value is never read.

Reply:
Oh, I just noticed, I will fix it.